docs/reference/experiment-config-reference.rst

```diff
@@ -321,18 +321,26 @@ defaults.
 
 .. _log-retention-days:
 
-``log_retention_days``
-======================
+``retention_policy``
+====================
+
+Optional. Defines retention policies for logs of all the trials of a given experiment. Parameters
+include:
```

Comment:
Optional. Defines retention policies for logs related to all trials of a given experiment. Parameters
include: